src/index.js

```diff
@@ -26,6 +26,12 @@ export function mapToRelative(currentFile, module) {
     return moduleMapped;
 }
 
+function resolve(filename) {
```

Comment:
Could you move `resolve()` above `mapToRelative` to fix the eslint error please? :)
Thanks!